### PR TITLE
Nightshade to 540

### PR DESCRIPTION
**Pull request by kennethgillen**
This PR builds on top of https://github.com/openmicroscopy/prod-playbooks/pull/4 - Not sure if this branch on the other branch is the right way to go about this. 

Merged in the .travis.yml from master to avoid conflicts, thanks to @jburel for the tip.

### CODE REVIEW

**Comment by joshmoore** [comment]
Should demo also be bumped to 5.4.0? i.e. this PR == "upgrade prod systems to 5.4"

**Comment by sbesson** [comment]
Should this now be closed and superseded by a PR bumping demo and nightshade to 5.4.1?

**Comment by kennethgillen** [comment]
@sbesson  - I'd like to get these in as a "state of 5.4.0", and get everything clean if possible before trying to add yet more here.

**Comment by joshmoore** [comment]
You'll need to try merging `origin/master` then and see what the conflicts are.

**Comment by kennethgillen** [comment]
Interesting, the merge appeared to work with no conflicts.

**Comment by kennethgillen** [comment]
Travis is green here, but the NS playbooks are excluded. Also, the filename is wrong, because I didn't rename the file in the .travis.yml in PR https://github.com/openmicroscopy/prod-playbooks/pull/8

**Comment by sbesson** [comment]
As discussed, :+1: from my side to get this state merged including the Travis check of the playbooks.

File: ome-dundeeomero.yml [hunk 1]
```diff
@@ -250,15 +253,14 @@
       omero.jvmcfg.system_memory: 17000
       omero.ldap.base: "{{ omero_server_ldap_base }}"
       omero.ldap.config: true
-      omero.ldap.new_user_group: default
       omero.ldap.urls: "{{ omero_server_ldap_urls }}"
       omero.mail.config: true
       omero.mail.from: "{{ omero_server_mail_from }}"
       omero.mail.host: "{{ omero_server_mail_host }}"
-      omero.new_user_group: "My Data"
+      omero.ldap.new_user_group: "My Data"
```
Comment (joshmoore) [inline ome-dundeeomero.yml]: This one will need to go along with a `bin/omero obj update ExperimenterGroup ... name="My Data"`

**Comment by joshmoore** [comment]
Barring the one comment, happy to have this merged ASAP.

File: ome-dundeeomero.yml (hunk 1 above)
Comment (kennethgillen) [reply ome-dundeeomero.yml]: @joshmoore - ordinarily, (for anyone following along) yes. But in the case of this particular server, we've renamed the group already. cf (Internal to OME) https://trello.com/c/BHPI0xPs

Comment (kennethgillen) [reply ome-dundeeomero.yml]: The potential cleanup step is to move users to "My Data" who are in the previous `omero.ldap.new_user_group`. Not sure of implications for data, i.e. if this needs to be a "move the data first, then change the group membership".

Comment (joshmoore) [reply ome-dundeeomero.yml]: >  But in the case of this particular server, we've renamed the group already. 

Which for me means, `Done` :+1:

**Pull request merged**